src/components/Widgets/StatPing.vue

```diff
@@ -109,6 +123,8 @@ export default {
           totalFailure: showNumAsThousand(service.stats.failures),
           lastSuccess: getTimeAgo(service.last_success),
           lastFailure: getTimeAgo(service.last_error),
+          chartHidden: this.showChart ? 0 : 1,
+          infoHidden: this.showInfo ? 0 : 1,
```

Comment:
Could just be `!this.showChart`, but this is fine also

Reply:
For some reason I didn't get the expected behaviour with boolean values, probably overlooked something...